src/data/STATUSES/layers/patch6.1.ts

```diff
@@ -12,6 +13,10 @@ export const patch610: Layer<StatusRoot> = {
 			duration: 20000,
 		},
 
+		//SAM 6.1 buff Change
+		THIRD_EYE: {duration: 4000},
+		KAITEN: {id: SHARED.UNKNOWN.id},
```

Comment:
Same here, just set it in full:
```suggestion
		KAITEN: SHARED.UNKNOWN,
```